packages/react-scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-scripts",
-  "version": "1.0.11",
+  "name": "@connected-home/react-scripts",
+  "version": "1.0.11.1",
```

Comment:
this looks unintended? did you mean to write `1.0.11-1` ? 

you can use `x.y.z` followed by `-` + `somestring` to create prerelease versions)
e.g `1.0.11-1`, `1.0.11-2`

see http://semver.org/#spec-item-9 for proper words on it